esy.json

```diff
@@ -125,23 +123,23 @@
     "@opam/stdio": "janestreet/stdio:stdio.opam#f006beb417ee568c612c3d89824cfb6e15a98568",
     "@opam/ppx_hash": {
       "version": "opam:v0.14.0",
-      "override" : {
+      "override": {
         "dependencies": {
           "@opam/ppx_sexp_conv": "*"
         }
       }
     },
     "@opam/ppx_assert": {
       "version": "opam:v0.14.0",
-      "override" : {
+      "override": {
         "dependencies": {
           "@opam/ppx_sexp_conv": "*"
         }
       }
     },
     "@opam/ppx_base": {
       "version": "opam:v0.14.0",
-      "override" : {
+      "override": {
```

Comment:
I guess this is formatter-related, so the version without space is the correct one?
Do we have a JSON formatter definer somewhere? To avoid this kind of noise in future PRs?

Reply:
I'm just using prettier by default

Reply:
So do I, but looks like we are not all using it.